### PR TITLE
feat: ALPHA-414 add sync endpoint to Union for Activity

### PR DESCRIPTION
**Pull request by Rtserg**

### CODE REVIEW

File: api/src/main/kotlin/com/rarible/protocol/union/api/controller/ActivityController.kt [hunk 1]
```diff
@@ -33,6 +38,22 @@ class ActivityController(
...
 
+    override suspend fun getAllActivitiesSync(
+        blockchain: BlockchainDto,
+        continuation: String?,
+        size: Int?,
+        sort: SyncSortDto?
+    ): ResponseEntity<ActivitiesDto> {
+        logger.info("Got request to get all activities sync, parameters: $blockchain, $continuation, $size, $sort")
+        val safeSize = PageSize.ACTIVITY.limit(size)
+        val activitySlice = router.getService(blockchain).getAllActivitiesSync(continuation, safeSize, sort)
```
Comment (Rtserg) [inline api/src/main/kotlin/com/rarible/protocol/union/api/controller/ActivityController.kt:49]: @denis-kilchichakov-rarible I walked around ActivitySourceSelectService this way, what do you think?

Comment (augur) [reply api/src/main/kotlin/com/rarible/protocol/union/api/controller/ActivityController.kt:49]: Looks like abstraction is leaking a bit. May be we should have separate Controller for sync operations? @sanufriev

Comment (Rtserg) [reply api/src/main/kotlin/com/rarible/protocol/union/api/controller/ActivityController.kt:49]: It looks like good idea, but I would do it in a separate ticket.

Comment (Rtserg) [reply api/src/main/kotlin/com/rarible/protocol/union/api/controller/ActivityController.kt]: @augur  We decided not to separate Controller, and put all the functionality to ActivityApiService, and throw UnsupportedOperationException in  ActivityElasticService.

**Pull request merged**